tests/README.md

```diff
@@ -77,20 +77,32 @@ The build script has the following parameters:
 Select `Ubuntu 14.04 LTS v1503 (beta with Docker support)`.
 The platform with *Docker support* means the tests will run in a VM.
 
-## Manually running the functional tests
```

Comment:
Ah, I realised why the doc is structured this way now - because it's scoped within the tests/ directory which only currently deals with functional tests. So really we should probably move this whole stanza into Documentation/hacking.md. But we can follow up with that - this LGTM for now. Thanks!